application/src/main/java/org/opentripplanner/transit/configure/TransitModule.java

```diff
@@ -14,6 +14,8 @@
 import org.opentripplanner.transit.service.TimetableRepository;
 import org.opentripplanner.transit.service.TransitService;
 import org.opentripplanner.updater.trip.TimetableSnapshotManager;
+import org.opentripplanner.updater.trip.TripPatternCache;
+import org.opentripplanner.updater.trip.TripPatternIdGenerator;
```

Comment:
These should be configured somewhere under the `o.o.updater` package.

Reply:
All of the real-time-related components? TimetableSnapshotManager, TimetableSnapshot, ReaTimeRaptorTransitDataUpdater..?

Reply:
I implemented this in https://github.com/opentripplanner/OpenTripPlanner/pull/6629/commits/0103c98a066468593e7d11a8fcc4398b6ca52e7e